leptos_dom/src/html.rs

```diff
@@ -276,6 +276,12 @@ impl ElementDescriptor for Custom {
 cfg_if! {
   if #[cfg(all(target_arch = "wasm32", feature = "web"))] {
     /// Represents an HTML element.
+    /// Beginner's tip: 
```

Comment:
There seems to be trailing whitespace here, which is why `cargo fmt` is failing. Would you mind running rustfmt on the code once?